crates/turbopack-ecmascript/tests/tree-shaker/analyzer/grouping/output.md

```diff
@@ -0,0 +1,520 @@
+    Item13;
+    Item13["export y"];
+    Item6 --> Item1;
+    Item6 --> Item2;
+    Item6 --> Item3;
+    Item6 --> Item4;
+    Item6 --> Item5;
+    Item7 --> Item6;
+    Item8 --> Item7;
+    Item9 --> Item8;
+    Item10 --> Item9;
+```
+# Phase 4
+```mermaid
+graph TD
+    Item1;
+    Item2;
+    Item3;
+    Item4;
+    Item5;
+    Item6;
+    Item7;
+    Item8;
+    Item9;
+    Item10;
+    Item11;
+    Item11["ModuleEvaluation"];
+    Item12;
+    Item12["export x"];
+    Item13;
+    Item13["export y"];
+    Item6 --> Item1;
+    Item6 --> Item2;
+    Item6 --> Item3;
+    Item6 --> Item4;
+    Item6 --> Item5;
+    Item7 --> Item6;
+    Item8 --> Item7;
+    Item9 --> Item8;
+    Item10 --> Item9;
+    Item12 --> Item9;
+    Item13 --> Item10;
+```
+# Final
+```mermaid
+graph TD
+    N0["Items: [ItemId(ModuleEvaluation)]"];
+    N1["Items: [ItemId(Export((&quot;x&quot;, #2), &quot;x&quot;))]"];
+    N2["Items: [ItemId(Export((&quot;y&quot;, #2), &quot;y&quot;)), ItemId(10, VarDeclarator(0))]"];
+    N3["Items: [ItemId(0, VarDeclarator(0))]"];
+    N4["Items: [ItemId(1, Normal)]"];
+    N5["Items: [ItemId(2, Normal)]"];
+    N6["Items: [ItemId(3, Normal)]"];
+    N7["Items: [ItemId(4, Normal)]"];
+    N8["Items: [ItemId(5, Normal)]"];
+    N9["Items: [ItemId(6, Normal)]"];
+    N10["Items: [ItemId(7, Normal)]"];
+    N11["Items: [ItemId(8, Normal)]"];
+    N1 --> N11;
+    N2 --> N11;
+    N8 --> N3;
+    N8 --> N4;
+    N8 --> N5;
+    N8 --> N6;
+    N8 --> N7;
+    N9 --> N8;
+    N10 --> N9;
+    N11 --> N10;
+```
+# Entrypoints
+
+```
+{
+    ModuleEvaluation: 0,
+    Export(
+        "y",
+    ): 2,
+    Export(
+        "x",
+    ): 1,
+}
+```
+
+
+# Modules (dev)
+## Part 0
+```js
+"module evaluation";
+
+```
+## Part 1
+```js
+import { x } from "__TURBOPACK_PART__" assert {
+    __turbopack_part__: 11
+};
+export { x as x };
+
+```
+## Part 2
+```js
+import { x } from "__TURBOPACK_PART__" assert {
+    __turbopack_part__: 11
+};
+export { y };
+const y = x;
+export { y } from "__TURBOPACK_VAR__" assert {
+    __turbopack_var__: true
+};
+
+```
+## Part 3
+```js
+let x = 1;
+export { x } from "__TURBOPACK_VAR__" assert {
+    __turbopack_var__: true
+};
+
+```
+## Part 4
+```js
+x = 2;
+export { x } from "__TURBOPACK_VAR__" assert {
+    __turbopack_var__: true
+};
+
+```
+## Part 5
+```js
+x = 3;
+export { x } from "__TURBOPACK_VAR__" assert {
+    __turbopack_var__: true
+};
+
+```
+## Part 6
+```js
+x = 4;
+export { x } from "__TURBOPACK_VAR__" assert {
+    __turbopack_var__: true
+};
+
+```
+## Part 7
+```js
+x = 5;
+export { x } from "__TURBOPACK_VAR__" assert {
+    __turbopack_var__: true
+};
+
+```
+## Part 8
+```js
+import { x } from "__TURBOPACK_PART__" assert {
+    __turbopack_part__: 3
+};
+import "__TURBOPACK_PART__" assert {
+    __turbopack_part__: 4
+};
+import "__TURBOPACK_PART__" assert {
+    __turbopack_part__: 5
+};
+import "__TURBOPACK_PART__" assert {
+    __turbopack_part__: 6
+};
```

Comment:
```suggestion
```

These are not necessary. It only need to import the last Write to a variable. And the declaration of it